doc/offline_maps.md

```diff
@@ -606,40 +781,83 @@ This special Vector Tile Data Source must be used: **PackageManagerTileDataSourc
       
       NTCompiledStyleSet *vectorTileStyleSet = [[NTCompiledStyleSet alloc] initWithAssetPackage:package];
     
-// 2. Create vector tile decoder using the styleset and update style parameters
+	 // 2. Create vector tile decoder using the styleset and update style parameters
       NTMBVectorTileDecoder* vectorTileDecoder = [[NTMBVectorTileDecoder alloc] initWithCompiledStyleSet:vectorTileStyleSet];
       [vectorTileDecoder setStyleParameter:@"lang" value:@"en"];
       
       // Optional: enable 3D elements
       [vectorTileDecoder setStyleParameter:@"buildings3d" value:@"YES"];
       [vectorTileDecoder setStyleParameter:@"markers3d" value:@"1"];
     
-// 3. Create tile data source from PackageManager
+	 // 3. Create tile data source from PackageManager
       NTTileDataSource* vectorTileDataSource = [[NTPackageManagerTileDataSource alloc] initWithPackageManager:_packageManager];
     
-// 4. Create vector tile layer, using previously created data source and decoder
+	 // 4. Create vector tile layer, using previously created data source and decoder
       NTVectorTileLayer* baseLayer = [[NTVectorTileLayer alloc] initWithDataSource:vectorTileDataSource decoder:self.vectorTileDecoder];
     
-// 5. Add vector tile layer as first layer
+	 // 5. Add vector tile layer as first layer
       [[mapView getLayers] insert:0 layer:baseLayer];
 
   {% endhighlight %}
   </div>
 
   <div class="Carousel-item js-Tabpanes-item--lang js-Tabpanes-item--lang--swift">
-  {% highlight swift %}COMING SOON...
+  {% highlight swift %}
+  
+        // 1. Create layer with vector styling
+        let styleBytes = NTAssetUtils.loadAsset("osmbright.zip")
+        let assetPackage = NTZippedAssetPackage(zip: styleBytes)
+        
+        var vectorTileDecoder: NTMBVectorTileDecoder?
+        
+        if (styleBytes != nil) {
+            // Create style set
+            let vectorTileStyleSet = NTCompiledStyleSet(assetPackage: assetPackage)
+            vectorTileDecoder = NTMBVectorTileDecoder(compiledStyleSet: vectorTileStyleSet);
+        }
+        
+        // 2. Create offline datasource from Package Manager
+        let dataSource = NTPackageManagerTileDataSource(packageManager: packageManager);
+        
+        let baseLayer = NTVectorTileLayer(dataSource: dataSource, decoder: vectorTileDecoder)
+        self.mapView?.getLayers().add(baseLayer)
 
   {% endhighlight %}
   </div>
+
+  <div class="Carousel-item js-Tabpanes-item--lang js-Tabpanes-item--lang--kotlin">
+  {% highlight kotlin %}
   
+        // 1. Create layer with vector styling
+        val styleBytes = AssetUtils.loadAsset("osmbright.zip")
+        val assetPackage = ZippedAssetPackage(styleBytes)
+
+        var vectorTileDecoder: MBVectorTileDecoder? = null
+
+        if (styleBytes != null) {
+            // Create style set
+            val vectorTileStyleSet = CompiledStyleSet(assetPackage)
```

Comment:
You use "styleset" one word above. It should be consistent throughout, if possible